src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -115,48 +116,48 @@
+(rf/reg-sub-raw
  ::catalogue-table-rows
- (fn [_ _]
-   [(rf/subscribe [::catalogue])
-    (rf/subscribe [:language])
-    (rf/subscribe [:rems.administration.administration/displayed-organization-ids])])
- (fn [[catalogue language displayed-organization-ids] _]
-   (->> catalogue
-        (administration/filter-by-displayed-organization displayed-organization-ids #(get-in % [:organization :organization/id]))
-        (mapv (fn [item]
-                {:key (:id item)
-                 :organization {:value (get-in item [:organization :organization/short-name language])}
-                 :name {:value (get-localized-title item language)
-                        :sort-value [(get-localized-title item language)
-                                     (- (time-coerce/to-long (:start item)))]} ; secondary sort by created, reverse
-                 :resource (let [value (:resource-name item)]
-                             {:value value
-                              :display-value [atoms/link nil
-                                              (str "/administration/resources/" (:resource-id item))
-                                              value]})
-                 :form (if-let [value (:form-name item)]
-                         {:value value
-                          :display-value [atoms/link nil
-                                          (str "/administration/forms/" (:formid item))
-                                          value]}
-                         {:value (text :t.administration/no-form)
-                          :sort-value "" ; push "No form" cases to the top of the list when sorting
-                          :display-value [text :t.administration/no-form]})
-                 :workflow (let [value (:workflow-name item)]
-                             {:value value
-                              :display-value [atoms/link nil
-                                              (str "/administration/workflows/" (:wfid item))
-                                              value]})
-                 :created (let [value (:start item)]
-                            {:value value
-                             :display-value (localize-time value)})
-                 :active (let [checked? (status-flags/active? item)]
-                           {:display-value [readonly-checkbox {:value checked?}]
-                            :sort-value (if checked? 1 2)})
-                 :commands {:display-value [:div.commands
-                                            [view-button (:id item)]
-                                            [modify-item-dropdown item]]}})))))
+ (fn [_db-sub]
+   (r/reaction
+    (let [language @(rf/subscribe [:language])
+          org-ids @(rf/subscribe [:rems.administration.administration/displayed-organization-ids])
+          filter-by-displayed-organizations (partial administration/filter-by-displayed-organization org-ids #(get-in % [:organization :organization/id]))]
+      (doall
```

Comment:
This feels like a step backwards. Should we just separate the `text` as function and `text` as component? Or if we used the text from a global var, we would not have to do this?

Also everything used to "just work" and https://github.com/day8/re-frame/pull/754 would just fix this?